server/jetstream_api.go

```diff
@@ -27,118 +28,121 @@ import (
 
 // Request API subjects for JetStream.
 const (
-	// JetStreamEnabled allows a user to dynamically check if JetStream is enabled for an account.
-	// Will return +OK on success, otherwise will timeout.
-	JetStreamEnabled = "$JS.ENABLED"
+	// JSApiInfo is for obtaining general information about JetStream for this account.
+	// Will return JSON response.
+	JSApiAccountInfo = "$JS.API.INFO"
 
-	// JetStreamInfo is for obtaining general information about JetStream for this account.
+	// JSApiCreateTemplate is the endpoint to create new stream templates.
 	// Will return JSON response.
-	JetStreamInfo = "$JS.INFO"
+	JSApiTemplateCreate  = "$JS.API.TEMPLATE.CREATE.*"
```

Comment:
we spoke now and then about maybe needing consumer templates to support dynamic creation of per account consumers, so `TEMPLATE` here might later not be specific enough

Reply:
They would need to be in the request for this template no?

Reply:
Understood after chatting, added STREAM